roofit/roofitcore/test/testRooCacheManager.cxx

```diff
@@ -60,3 +62,45 @@ TEST(RooCacheManager, TestSelectFromArgSet)
    EXPECT_TRUE(sel12.containsInstance(vars[4]));
    EXPECT_TRUE(sel21.containsInstance(vars[6]));
 }
+
+// This test doesn't work on Windows because the trick with defining protected as public doesn't work.
```

Comment:
Outdated comment? I don't see any `#define`, neither `#ifdef MSVC`

Reply:
Right, I need to remove this comment.